aces/Loris
# [media] fixes permissions. Allows media_read to download, renames "Delete" to "Hide".

## Brief summary of changes

Allows the `media_read` permission to actually download (see) the files. The permission in this descriptions stats `Media: View/Download Candidate Media Files` but was not actually allowing to download (see) the documents.

`media_write` was substituted by `media_read` in the corresponding cases.

The permission `Media: Edit/Upload/Delete Candidate Media Files` is renamed to `Media: Edit/Upload/Hide Candidate Media Files` since we are not able to Delete files but to Hide they.
 

#### Testing instructions (if applicable)

1. Go to the media module with a user having only the `media_read` permission.
2. You should be now capable of downloading (seeing) the files.
3. As admin go to edit candidate and see the permission `Media: Edit/Upload/Delete Candidate Media Files` is renamed to `Media: Edit/Upload/Hide Candidate Media Files` .
4. The rest of the functionalities in the module should be rest unchangeable. 

#### Link(s) to related issue(s)

* Resolves #9162
